website/static/js/comment.js

```diff
@@ -298,7 +298,12 @@ var CommentModel = function(data, $parent, $root) {
             'gravatarUrl': userData.links.profile_image
         };
     } else {
-        self.author = self.$root.author;
+        self.author = {
+            'id': null,
+            'urls': {'profile': ''},
+            'fullname': 'Deactivated user',
```

Comment:
The problem is that the data doesn't come through with deactivated users. I'll look further into it, but for now `A deactivated user`